ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -83,30 +84,65 @@
           .put(Schema.Type.FLOAT64, Schema.OPTIONAL_FLOAT64_SCHEMA)
           .build();
 
+  private static final ImmutableMap<String, String> TYPE_MAP =
```

Comment:
Can we rename this to something like `SCHEMA_TYPE_NAME_TO_SQL_TYPE`? Now that we've got multiple "type maps" in this file the generic name is a little confusing.

Reply:
Thanks. I changed it.